modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -464,3 +452,5 @@ export default class NestedTabs extends Base {
 		};
 	}
 }
+
+window.elementorModules.frontend.handlers.NestedTabs = NestedTabs;
```

Comment:
For now this will cause problems for the Mega Menu.

We will need to create an experiment for the Mega Menu to conditionally load the old or new NestedTabs module for BC reasons (we can remove this experiment after 2 versions).